doc/data_science/ai_architecture/parallel_processing.rst

```diff
@@ -0,0 +1,29 @@
+Parallel Processing for AI/ML
+===============================
+
+Parallel Processing in Exasol means that computations are distributed across multiple nodes in the database cluster, allowing large datasets to be processed simultaneously rather than sequentially.
```

Comment:
```suggestion
Parallel Processing in Exasol means that computations are distributed across all nodes in the database cluster, allowing large datasets to be processed simultaneously rather than sequentially.
```

That a node doesn't do anything is usually the exception, which only happens with poorly distributable tasks, such as virtual schemas or set UDFs with less groups than nodes.